modules/pulsepointLiteBidAdapter.js

```diff
@@ -10,6 +10,10 @@ const NATIVE_DEFAULTS = {
   ICON_MIN: 50,
 };
 
+const DEFAULT_BID_TTL = 20;
```

Comment:
this value is in seconds - is that right? Just double checking.

Reply:
Yes. ttl is in seconds, according to docs here - http://prebid.org/dev-docs/bidder-adapter-1.html.

Reply:
cool. LGTM.